src/PotentialTypes/EmpiricalPotentials/bm.jl

```diff
@@ -1,55 +1,48 @@
 ##############################   Born-Mayer  ###################################
-# mutable struct BornMayer <: EmpiricalPotential
-#     A::Float64
-#     ρ::Float64
-# end
+struct BornMayer <: EmpiricalPotential
+    A::Float64
+    ρ::Float64
+    rcutoff :: AbstractFloat
+end
 
-# function BornMayer()
-#     #ToDO
-#     return BornMayer(1.0, 1.0)
-# end
+get_trainable_params(bm::BornMayer) = Parameter{:A,:ρ}((bm.A, bm.ρ))
 
-# function get_trainable_params(bm::BornMayer)
-#     return (A = bm.A, ρ = bm.ρ)
-# end
+get_nontrainable_params(bm::BornMayer) = Parameter{:rcutoff}((bm.rcutoff))
 
-# function get_nontrainable_params(lj::BornMayer)
-#     p = Parameter{}(())
-#     return p
-# end
 
 # ##############################   Energy  ###################################
 
-# function potential_energy(r::Vector{<:Real}, p::BornMayer)
-#     return p.A * exp(-norm(r) / p.ρ)
-# end
+function potential_energy(R::AbstractFloat, bm::BornMayer)
+    bm.A * exp(-R / bm.ρ)
+end
 
 
 # ##############################   Force   ###################################
 
-# function force(r::Vector{<:Real}, p::BornMayer)
-#     d = norm(r) 
-#     return p.A /p.ρ * exp(-d / p.ρ ) .* r ./ d
-# end
+force(r::SVector{3,<:AbstractFloat}, bm::BornMayer) = force(r, norm(r), bm)
```

Comment:
since we have an identical function in the LJ case, I'd suggest extracting this as the more general
```julia
force(r::SVector{3,<:AbstractFloat}, p::EmpiricalPotential) = force(r, norm(r), p)
```

Reply:
unless you can think of any case where we wouldn't need the norm of the distance vector